chia/farmer/farmer_api.py

```diff
@@ -517,14 +520,19 @@ async def request_signed_values(self, full_node_request: farmer_protocol.Request
             self.farmer.log.error(f"Do not have quality string {full_node_request.quality_string}")
             return None
 
-        (plot_identifier, challenge_hash, sp_hash, node_id) = self.farmer.quality_str_to_identifiers[
-            full_node_request.quality_string
-        ]
+        (
+            plot_identifier,
+            challenge_hash,
+            sp_hash,
+            node_id,
+            filter_prefix_bits,
+        ) = self.farmer.quality_str_to_identifiers[full_node_request.quality_string]
```

Comment:
if you already store the `filter_prefix_bits` here, in the `self.farmer.quality_str_to_identifiers` dictionary. Do you really need to bounce it against the harvester at all?